octo-core/lib/octocore.rb

```diff
@@ -3,17 +3,22 @@
 require 'logger'
 
 require 'octocore/version'
-require 'octocore/utils'
 require 'octocore/config'
 require 'octocore/models'
 require 'octocore/counter'
+# require 'octocore/email'
```

Comment:
We need it. It is the functionality Ravin Bhaiya made. Although it is leading to weird error " **mandril** not found", so probably once integrated with master this error has to be looked at.